### ISSUE TITLE
Description Review testing (Single Balloon Scenarios)

### ISSUE THREAD

**Issue by terracoda** [phetsims/balloons-and-static-electricity#215]
Opening this issue to record results from description review.
@jessegreenberg and @emily-phet, @terracoda will keep adding to this list.

Sim Version [1.1.6-dev1](http://www.colorado.edu/physics/phet/dev/html/balloons-and-static-electricity/1.1.6-dev.1/balloons-and-static-electricity_en.html?hideChargeControls)

System: 2015 MacBook Pro Sierra (10.2.1?) with Safari and VO

**EDIT: @jessegreenberg will distribute this into a list of issues and check boxes as they open.**
**Unfinished Random List of Issues:**
- [x] Scene Summary - need to use shorter charge description. Net charge only for all three objects. @terracoda to provide example.
- [x] Initial position of the Yellow Balloon is incorrect (also in Scene Summary). Should be "In center of Play Area, evenly between sweater and wall."
- [x] Hmm, seems Play Area grid description is inconsistent. There seems to be more than one "center", and exact center, and a center that I defined as the starting position of the balloon, "evenly between sweater and wall." @terrcoda, to discuss with @jessegreenberg and @emily-phet.
- [x] Description for "Near wall" is incorrect (perhaps related to above). Currently when near wall, I get "In upper right edge of Play Area." It should just be "Near wall." "Near wall" should replace the incorrect description when the user uses the JN hotkeys, as well. 
- [x] Interactive alerts readable and accessible in the PDOM (Reset all, Balloon reset, Wall removed, Wall added). I was able to read these alerts in PDOM. They should remain hidden at all times in PDOM.
- [x] Release alerts: End position on sweater missing. @terracoda to provide examples.
- [x] Release alert: Missing color of balloon. We will need this for two-balloon scenarios.
- [x] Grab Balloon button alert: Change "Yellow Balloon" to "Yellow balloon grabbed". This will read out as "Yellow balloon grabbed, web application." At least that is what it should do in VO. Will need to test other screen readers.
- [x] Remove "Grabbed" from the beginning of the upon-grabbed interaction alert. I think the above change will help communicate the "grabbed" state better. May need "Yellow/Green balloon" at beginning of description.
- [x] Upon grabbing the balloon, the initial description containing position, charge and, instructions rarely fires (VO in Safari). Need to test further.
- [x] "At top right arm", better as "On right shoulder." @terracoda to provide updated sweater locations.
- [x] On left arm, but not on charges, I hear "Near Sweater" and then a combined description when I moved up, "No change in charges, more charges up and to the left. Near sweater." Likely related to grid inconsistency.

**Comment by jessegreenberg** [comment]
Additional issues from discussion with @terracoda:

 - When the balloon is not yet on the sweater and the sweater rubs up or down, we hear "No change in charges..." alert.  We shouldn't hear this until the ballloon is actually on the sweater and can pick up charges.

 - Charge item in scene summary should be reduced to not include charge comparisions:
    - example: Yellow baloon has a net negative charge.  Sweater has a net negative charge.  Wall has a net neutral charge.
    - When all charges are the same, summarize like:
    - "Yellow balloon, sweater and wall have net neutral charge."
    - "Yellow balloon, Green balloon, sweater, and wall have net neutral charge."

- The sim is currently using an old version of the play area grid.  The grid will need to be updated to match the current design in https://docs.google.com/spreadsheets/d/1BiXFN2dRWfsjqV2WvKAXnZFhsk0jCxbnT0gkZr_T5T0/edit?ts=568067c0#gid=102492439

Regarding:
- Interactive alerts readable and accessible in the PDOM (Reset all, Balloon reset, Wall removed, Wall added). I was able to read these alerts in PDOM. They should remain hidden at all times in PDOM.

    - We will try hiding the alerts until an alert needs to be announced.  This could potentially enable them for a single alert, and keep them hidden from the virtual cursor.

- When the balloon is 'grabbed' the label should change to "Yellow Balloon Grabbed". Currently "Grabbed" is added in the description, it should be added in the label.

 - >Upon grabbing the balloon, the initial description containing position, charge and, instructions rarely fires (VO in Safari). Need to test further.
    - This might be related to https://github.com/phetsims/balloons-and-static-electricity/issues/183

- Sonnification seems broken. It should be looked into.

-

**Comment by jessegreenberg** [comment]
Also discussed:

The sim alerts are tailored to hearing what happens when an interaction is completed (key is released).

When interacting with the mouse, we should hear aria-live alerts as the balloon enters new areas in the play area.  This should reduce the frequency of alerts so that the user doesn't hear too many alerts as they drag continuously with mouse or finger.

**Comment by jessegreenberg** [comment]
@terracoda I went through and made individual issues for the topics discussed here.  Can we close this issue?

**Comment by terracoda** [comment]
Thanks @jessegreenberg! Closing issue.

**Comment by terracoda** [comment]
@jessegreenberg regarding discussion about mouse and touch alerts (https://github.com/phetsims/balloons-and-static-electricity/issues/215#issuecomment-268375553), the goal for design might be to create a less verbose mode for keyboard users, as well. This might be helpful for all users needed some, but not all text output.
